src/fetchtastic/download/cli_integration.py

```
@@ -0,0 +1,780 @@
+                logger.debug(f"Using existing active prerelease: {remote_dir}")
+                return False, [remote_dir]
+            elif latest_active_dir:
+                # Latest active prerelease found remotely but not local
+                remote_dir = latest_active_dir
+                logger.debug(f"Found remote active prerelease: {remote_dir}")
+            else:
+                # No active prerelease found, fall back to directory scanning
+                remote_dir = None
+                logger.debug("No active prerelease found in commit history")
+
+        except Exception as exc:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)